internal/controller/spec/template/yaml/Templates.go

```diff
@@ -152,12 +152,10 @@ data:
     #echo "$dt - Running: psql -v ON_ERROR_STOP=1 --username $POSTGRES_USER --dbname $POSTGRES_DB ...";
 
     #psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
-      #CREATE DATABASE $customDatabaseName;
-      #\connect $customDatabaseName;
-      #CREATE USER $customUserName WITH PASSWORD '$POSTGRES_MYAPP_PASSWORD';
-      #GRANT SELECT, INSERT, UPDATE, DELETE, TRUNCATE, REFERENCES, TRIGGER ON ALL TABLES IN SCHEMA public TO $customUserName;
-      #GRANT USAGE, SELECT, UPDATE ON ALL SEQUENCES IN SCHEMA public TO $customUserName;
-      #GRANT USAGE, CREATE ON SCHEMA public TO $customUserName;
+    #CREATE USER $customUserName WITH PASSWORD '$POSTGRES_MYAPP_PASSWORD';
+    #CREATE DATABASE $customDatabaseName;
+    #\connect $customDatabaseName;
+    #GRANT ALL ON SCHEMA public TO $customUserName;
```

Comment:
what this change is intended for?

Reply:
Oh yes, this is autogenerated, I run a make all, which does a make generate and this showed up. 
I think last time it was changed:
https://github.com/tetrateio/kubegres/commit/6f4f3c01cd7f29daba6d76f2fa33aa3e6a35abe4#diff-90e4e58ab89abdeab871e3d899e938c306f9ac97bc12da9fa9883169649ababaR88
, but then forgot to run make generate or add the changes.